HousingSearchListener/V1/Factories/QueryableFactories/TenuresFactory.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Hackney.Shared.HousingSearch.Gateways.Models.Assets;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using HousingSearchListener.V1.Domain.Tenure;
+using HousingSearchListener.V1.Factories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QueryableTenuredAsset = Hackney.Shared.HousingSearch.Gateways.Models.Tenures.QueryableTenuredAsset;
+
+namespace HousingSearchListener.V1.Factories.QueryableFactories
+{
+    public class TenuresFactory : ITenuresFactory
+    {
+        public QueryableAssetTenure CreateAssetQueryableTenure(TenureInformation tenure)
+        {
+            if (tenure is null) throw new ArgumentNullException(nameof(tenure));
+
+            return new QueryableAssetTenure()
+            {
+                Id = tenure.Id,
+                EndOfTenureDate = tenure.EndOfTenureDate,
+                PaymentReference = tenure.PaymentReference,
+                StartOfTenureDate = tenure.StartOfTenureDate,
+                Type = tenure.TenureType.Description
+            };
+        }
+
+        public QueryableTenure CreateQueryableTenure(TenureInformation tenure)
```

Comment:
Hi Hanna, I believe this method exists also in the ESEntityFactory - could you confirm if it is actually needed in both places?